src/sql/plot.py

```diff
@@ -268,10 +268,9 @@ def _min_max(conn, table, column, with_=None, use_backticks=False):
 """
     if use_backticks:
         template_ = template_.replace('"', "`")
-
+        table = table.replace('"', "`")
```

Comment:
why do we need this? it might break stuff in other dbs

Reply:
looked from sqlglot and where it comes back with use backticks this should apply to `"schema"."table"`  and spark was erroring on this with  `"schema"."table"` for spark and I suspect a few more db that are currently marked `XFAIL` for `--schema`.

Reply:
Did a reread through, `schema` and `table` are considered identifiers, which is what the `use_backticks` checks, some dialects support `"` and ` and some will only support either or. As we've already checked if the dialect can use ` then this is safe for all dialects that support backticks. The previous method could fail for dbs where schema is passed in and `"` is not supported as an identifier. 

There are probably other currently failing dialects due to this.

https://github.com/tobymao/sqlglot/blob/55735f5ec546dcf1b663058a72fbea47c280b487/sqlglot/tokens.py#L552

Reply:
Function here already does it for calls without `schema`, last step of calls performs the ` backtick conversion. 
https://github.com/ploomber/jupysql/blob/4fda1656df105b296cf684b2fab92f47802e96ee/src/sql/plot.py#L235

Maybe it would be better to refactor these calls to:
```python
   _table = enclose_table_with_double_quotations(f'{schema}.{_table}') if schema else enclose_table_with_double_quotations(table, conn)

```

Reply:
ok, I understand why the change is required. let's keep it as is and please open an issue so we can think of a long-term solution

Reply:
https://github.com/ploomber/jupysql/issues/970